checklist.bs

```diff
@@ -239,6 +258,15 @@ host system with a **valid access token** and the syntax specified in
 The test target host system must respond with 200 OK and a JSON body containing the PCF with the
 requested `pfId`, as specified in [[DXP#api-action-get-response]].
 
+The test target host system must respond with
+- 200 OK and a JSON body containing the PCF with the
+    requested `pfId`, as specified in [[DXP#api-action-get-response]], or
+
+- 400 BAD REQUEST and a JSON body containing an error, as specified in
```

Comment:
Likewise (see prev. comment for Test Case 004)

Reply:
And see above as well ;)